app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeViewModel.kt

```diff
@@ -24,6 +24,7 @@ interface Output : BaseOutput {
     val trendingCoins: StateFlow<List<CoinItemUiModel>>
 }
 
+@Suppress("FunctionNaming", "LongMethod")
```

Comment:
- what's wrong with function naming issue with VM class?
- We should not `Suppress` LongMethod at class level, better to move it to specific method instead

Reply:
@Wadeewee Fixed in my PR https://github.com/nimblehq/jetpack-compose-crypto/pull/40/files#diff-de36febb4f40e6a521e038139a90143d0c51fd831d0eb44abbbe9533ececf4f3L27